isis/src/hayabusa2/apps/hyb2pds4gen/hyb1pds4gen.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="hyb1pds4gen" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
+
+  <brief>
+    Convert Hayabusa1 image from from ISIS3 cube to PDS4 format
+  </brief>
+
+  <description>
+   This programs exports an ISIS3 Haybusa cube to a PDS4 product. 
+   If available, Instrument and  Mapping information will be written to the detached 
+   output PDS4 formatted xml label file.
+    <p>
+       Note that not all require keyword values in the generated PDS4 labels can be determined automatically

```

Comment:
Typo:  require -> required

Reply:
I removed all the docs with the plan of re-writing them when I more of it done later.